README.md

```diff
@@ -60,11 +60,12 @@ Chapter 2 Introduction
 Chapter 3 Data visualisation 
 
 - Luke Morris: [VIDEO](https://youtu.be/TuWkMvQbYPI) | [Gina Reynolds' ggplot flipbook](https://evamaerey.github.io/ggplot_flipbook/ggplot_flipbook_xaringan.html)
+
 - Kamran Khan: [VIDEO](https://youtu.be/1Kl-Ma2Ajk8) | [SLIDES]()
 
 Chapter 4 Workflow: basics 
 
-- 
+- Erick Knackstedt: [VIDEO](https://youtu.be/utmMd8QEq7Y) | [SLIDES](https://morrisluke.github.io/bookclub-R_for_Data_Science/Presentations/Week03/Cohort1/Chapter4Slides.html) 
```

Comment:
Can you point this at r4ds.github.io instead of morrisluke.github.io? Best to have it live all in the same base repo :)

Reply:
Don't worry if it doesn't work right away, I'll worry about that :D